### ISSUE TITLE
privmsg parser chokes on nicks with backticks

### ISSUE THREAD

**Issue by jfhbrook** [gf3/IRC-js#2]
Hey there! Sorry, got another one. If someone has a backtick in their nick, then the parser throws an error:

```
[ERROR] Parsing ':jesusabd`work!~josh@77-116.trans.uaf.edu PRIVMSG #stackvm :!wx 99775'
```

I found out that this only happens if one has a backtick in their nick. These are relatively uncommon on freenode, but extremely common on other networks--for example, gamesurge.

I tried to modify the regexp in Grammar.Person in parser_grammar.js from

```
$(/^[-\[\]|_\w\.]+/, "nick")
```

to

```
$(/^[-`\[\]|_\w\.]+/, "nick")
```

but no dice. I asked some javascripters on irc about this regexp (I'm terrible at them) and they thought that there should be a trailing dollar sign. Idk--ymmv.

**Comment by klocatelli** [comment]
I'm sure you've long solved this... however the bug still exists in the latest version of the library as of this post, so I'll put the uber-simple fix that I'm using:

<pre>
In parser_grammar.js
Change: $([$(/^[-\[\]|_\w\.]+/, "nick")
To this: $([$(/^[-\[\]|_\w\.\`]+/, "nick")
</pre>

**Comment by gf3** [comment]
Fixed in ebbdc543940b394ab60473fb04fdaf38bdbca7ca

Thanks guys!
